pkg/tools/spanhelper/span_helper.go

```diff
@@ -259,10 +281,10 @@ func (s *spanHelper) Context() context.Context {
 	return s.ctx
 }
 
-// FromContext - return span helper from context and if opentracing is enabled start new span
+// FromContext - return span helper from context and if opentelemetry is enabled start new span
 func FromContext(ctx context.Context, operation string) (result SpanHelper) {
-	if jaeger.IsOpentracingEnabled() {
-		newSpan, newCtx := opentracing.StartSpanFromContext(ctx, operation)
+	if opentelemetry.IsOpentelemetryEnabled() {
+		newCtx, newSpan := otel.Tracer("").Start(ctx, operation)
```

Comment:
Please check why "". Could we declare tracer with nsm? and use it?

Reply:
Yep, we can. Changed to "NSM"